### PR TITLE
doc: add SetReportingPeriod recommendations

### PR DESCRIPTION
**Pull request by jadekler**

### CODE REVIEW

File: stats/view/worker.go [hunk 1]
```diff
@@ -113,11 +113,15 @@ func record(tags *tag.Map, ms interface{}) {
...
-}
+};
 
 // SetReportingPeriod sets the interval between reporting aggregated views in
-// the program. If duration is less than or
-// equal to zero, it enables the default behavior.
+// the program. If duration is less than or equal to zero, it enables the
+// default behavior.
+//
+// Note: each exporter makes different promises about what the lowest supported
```
[the remark this answers is not in the document]
Comment (odeke-em) [reply stats/view/worker.go]: @songy23 I see you initially mailed https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/pull/37 but closed it. Will you perhaps be making an update in light of this new PR?

[the remark this answers is not in the document]
Comment (songy23) [reply stats/view/worker.go]: Yes, the Stackdriver example is already updated in https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/pull/38.

File: stats/view/worker.go [hunk 2]
```diff
@@ -113,11 +113,15 @@ func record(tags *tag.Map, ms interface{}) {
 		ms: ms.([]stats.Measurement),
 	}
 	defaultWorker.c <- req
-}
+};
```
[the remark this answers is not in the document]
Comment (jadekler) [reply stats/view/worker.go]: Done

**Comment by jadekler** [comment]
PTAL

**Pull request merged**